lib/liquid_voting/tracers.ex

```diff
@@ -0,0 +1,14 @@
+defmodule LiquidVoting.Tracers do
+  require OpenTelemetry.Tracer, as: Tracer
+
+  # Helper functions for our use of OpenTelemetry.Honeycomb.Exporter
+
+  # Sets general attributes to send in Tracer.with_span block.
+  def set_attributes(env, vars) do
+    Tracer.set_attributes([
+      {:action, env.function},
```

Comment:
I think it might be more useful to name this explicitly as `:function`. We could then develop this to actually include the module name + function name (and then name the field something like `:mod/fun`. This would make more sense in the cases where we also wrap tracers around smaller sections of function to get more granular info. on things like db queries, etc.